.github/workflows/tests.yaml

```diff
@@ -37,6 +33,11 @@ jobs:
         run: poetry install --with docs,tutorial
       - name: Build coverage file
         run: |
-          poetry run pytest --cache-clear --cov=hamilflow tests/ > pytest-coverage.txt
-      - name: Comment coverage
-        uses: coroo/pytest-coverage-commentator@v1.0.2
+          poetry run pytest --cache-clear --junitxml=pytest.xml --cov=hamilflow tests/ > pytest-coverage.txt
+      - name: Pytest coverage comment
+        uses: MishaKav/pytest-coverage-comment@v1.1.51
+        with:
+          pytest-coverage-path: ./pytest-coverage.txt
+          junitxml-path: ./pytest.xml
+          create-new-comment: false
+          title: Coverage Report (${{ github.event.pull_request.head.sha || github.sha }}, ${{ matrix.python-version }}, ${{ matrix.os }})
```

Comment:
I mean to print the poetry version

Reply:
not sure if that is gonna be useful.

Reply:
As long as it's synced with the lock file, I am fine

Reply:
`poetry` is sort of global on each of our machine and its version in Actions is set manually.
I guess we have to agree on some conventions on this problem. 
For now, I would say, we don't deal with it unless we run into problems?

Reply:
Fine by me. Poetry should be back compatible enough (hopefully, being such a high profile tool), at least for minor versions.